<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarPopUp</class>
 <widget class="QWidget" name="CalendarPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">alternate-background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="title_LA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>58</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="title_LE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="description_LA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="description_LE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_period_LA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Time period</string>
   </property>
  </widget>
  <widget class="QLabel" name="from_LA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="from_DE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="displayFormat">
    <string>M/d/yy</string>
   </property>
  </widget>
  <widget class="QLabel" name="till_LA">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>TIll</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="till_DE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="displayFormat">
    <string>M/d/yy</string>
   </property>
  </widget>
  <widget class="QComboBox" name="repeat_CB">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>Never</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Daily</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Weekly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monthly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Annually</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="save_PB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="repeat_LA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abort_PB">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-160</x>
     <y>0</y>
     <width>611</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.715909, stop:0 rgba(0, 0, 0, 9), stop:0.375 rgba(0, 0, 0, 50), stop:0.835227 rgba(0, 0, 0, 75));
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>58</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/images/BankLogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>title_LA</zorder>
  <zorder>title_LE</zorder>
  <zorder>description_LA</zorder>
  <zorder>description_LE</zorder>
  <zorder>time_period_LA</zorder>
  <zorder>from_LA</zorder>
  <zorder>from_DE</zorder>
  <zorder>till_LA</zorder>
  <zorder>till_DE</zorder>
  <zorder>repeat_CB</zorder>
  <zorder>save_PB</zorder>
  <zorder>repeat_LA</zorder>
  <zorder>abort_PB</zorder>
  <zorder>label</zorder>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
